v3/src/components/slider/inspector-panel/slider-settings-panel.tsx

```diff
@@ -23,16 +24,53 @@ export const SliderSettingsPalette =
   const handleMultiplesOfChange = (value: string) => {
     const multipleOf = parseFloat(value)
     if (isFinite(multipleOf)) {
-      sliderModel.setMultipleOf(multipleOf)
+      sliderModel.applyModelChange(() => {
+        sliderModel.setMultipleOf(multipleOf)
+      }, {
+        undoStringKey: "DG.Undo.slider.changeMultiples",
+        redoStringKey: "DG.Redo.slider.changeMultiples",
+        log: logStringifiedObjectMessage("sliderMaxPerSecond: %@",
```

Comment:
```suggestion
        log: logStringifiedObjectMessage("sliderMultiplesOf: %@",
```
or whatever the appropriate spelling is for v2 compatibility.